File: BHOI_/2013/Identicna jaja/grader.pas
Program stub;

uses
    jaje, jajelib;

var
    n, k, rezultat: LongInt;
    fifoin, fifoout: Text;

begin
    assign(fifoin, argv[2]);
    assign(fifoout, argv[1]);
    reset(fifoin);
    rewrite(fifoout);
    readln(fifoin, n, k);
    Inicijalizacija(fifoin, fifoout);

    rezultat := PronadjiSprat(n,k);
    writeln(fifoout,'1 ',rezultat);
    flush(fifoout);
    close(fifoin);
    close(fifoout);
    Halt(0);
end.

File: BHOI_/2013/Identicna jaja/grader.c
#include <stdio.h>
#include <stdlib.h>

int PronadjiSprat(int n, int k);

FILE *fifo_in, *fifo_out;

int BaciJaje(int i) {
	int r, tmp;
	tmp = fprintf(fifo_out, "0 %d\n", i);
	fflush(fifo_out);
	tmp = fscanf(fifo_in, "%d", &r);
	if (tmp == tmp)
	    return r;
}

int main(int argc, char **argv) {
	fifo_in = fopen(argv[2], "r");
	fifo_out = fopen(argv[1], "w");

	int n, k, tmp, rezultat;
	tmp = fscanf(fifo_in, "%d %d", &n, &k);
	rezultat = PronadjiSprat(n, k);
	tmp = fprintf(fifo_out, "1 %d\n", rezultat);
	fflush(fifo_out);

	fclose(fifo_in);
	fclose(fifo_out);
	
	if (tmp == tmp)
		return 0;
}
